google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -114,11 +152,16 @@ public void sendEvent(
         // I think 'virtual' indicates these don't correspond to real web pages.
         postData.add(new BasicNameValuePair(IS_VIRTUAL_KEY, STRING_TRUE_VALUE));
         if (eventLabel != null) {
-          // Event metadata are passed as a (virtual) page title.
-          String virtualPageTitle = eventLabel + "=" + (eventValue != null ? eventValue : "null");
-          postData.add(new BasicNameValuePair(PAGE_TITLE_KEY, virtualPageTitle));
+          Map<String, Integer> eventMetaDataMap = Maps.newHashMap();
+          eventMetaDataMap.put(METADATA_ESCAPER.escape(eventLabel), eventValue);
+          metadataStringBuilder =
+              METADATA_JOINER.appendTo(
+                  // I feel like I shouldn't have to do the ',' append here but for some reason
+                  // I do.  Bug?
+                  metadataStringBuilder.append(','), eventMetaDataMap);
```

Comment:
Yeah, I was avoiding recomputing the prefix for each analytics request. Probably a small cost. But I've seen a bug actually.